src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -180,10 +181,12 @@ const OrderRow: React.FC<Props> = props => {
   const unlimitedTime = useMemo(
     () =>
       dateFromBatchId(order.validUntil).getTime() >=
-      addTimeToDate(new Date(), MIN_UNLIMITED_SELL_ORDER_EXPIRATION_TIME, 'minute').getTime(),
+      addMinutes(Date.now(), MIN_UNLIMITED_SELL_ORDER_EXPIRATION_TIME).getTime(),
```

Comment:
not from this PR, but I think we don't need `getTime()` to compare two dates

also, not from this PR, but I would say MIN_UNLIMITED_SELL_ORDER_EXPIRATION_TIME of 1 year is too litle. I was thinking in an arbitrary high amount. We'll set the limit to the max value, so probably it will be an epoc of 2^256-1  I'm ok using for the comparison something like 100 years too

Reply:
It's likely that we will need this somewhere else, and it would make it more easy to test
Would it make sense to move the check to the utils/helpers?

also, this way, we don't add a direct dependency with `date-fns` in the UI

Reply:
Not a big deal, addressed all changes in this PR